src/logic/windowcheck/WindowCheck.swift

```diff
@@ -0,0 +1,175 @@
+//
+// Created by Zachary Wander on 8/14/22.
+// Copyright (c) 2022 lwouis. All rights reserved.
+//
+
+/**
+ The collection of all window checkers.
+ If a new check is created, its instance should be added here.
+ */
+let windowChecks: [WindowCheck] = [
+    JetBrainsWindowCheck(),
+    IINAWindowCheck(),
+    KeyNoteWindowCheck(),
+    OpenBoardWindowCheck(),
+    AdobeAuditionWindowCheck(),
+    BooksWindowCheck(),
+    WorldOfWarcraftWindowCheck(),
+    BattleNetBootstrapperWindowCheck(),
+    DrBetotteWindowCheck(),
+    DVDFabWindowCheck(),
+    SanGuoShaAirWDWindowCheck(),
+    SteamWindowCheck(),
+    FirefoxFullscreenVideoWindowCheck(),
+    VLCFullscreenVideoWindowCheck(),
+    AndroidEmulatorWindowCheck(),
+    ColorSlurpWindowCheck()
+]
```

Comment:
I'm not too familiar with Swift so there may be a better way collect all the subclasses.